adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -688,21 +687,6 @@ def reconnect(self, resub_topics=True):
                 feed = subscribed_topics.pop()
                 self.subscribe(feed)
 
-    def loop_forever(self):
```

Comment:
Self-note that once this PR is approved and tested, we'll need to file an issue in the following libraries and remove the deprecated `loop_forever` method:

CircuitPython Adafruit IO

- [ ] https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/blob/master/adafruit_io/adafruit_io.py#L231

CircuitPython GCP IoT Core

- [ ] https://github.com/adafruit/Adafruit_CircuitPython_GC_IOT_Core/blob/master/adafruit_gc_iot_core.py#L222

Reply:
these issue have been opened per request.
See [issue for Adafruit_CircuitPython_AdafruitIO ](https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/issues/40) & 
[issue for Adafruit_CircuitPython_GC_IOT_Core ](https://github.com/adafruit/Adafruit_CircuitPython_GC_IOT_Core/issues/13)

Reply:
Wow, thanks!